crates/sp-domains-fraud-proof/src/host_functions.rs

```diff
@@ -120,13 +155,62 @@ where
                 .map(|block_randomness| {
                     FraudProofVerificationInfoResponse::BlockRandomness(block_randomness)
                 }),
-            FraudProofVerificationInfoRequest::DomainTimestampExtrinsic(doman_id) => self
-                .derive_domain_timestamp_extrinsic(consensus_block_hash, doman_id)
+            FraudProofVerificationInfoRequest::DomainTimestampExtrinsic(domain_id) => self
+                .derive_domain_timestamp_extrinsic(consensus_block_hash, domain_id)
                 .map(|domain_timestamp_extrinsic| {
                     FraudProofVerificationInfoResponse::DomainTimestampExtrinsic(
                         domain_timestamp_extrinsic,
                     )
                 }),
+            FraudProofVerificationInfoRequest::DomainBundleBody {
+                domain_id,
+                bundle_index,
+            } => self
+                .get_domain_bundle_body(consensus_block_hash, domain_id, bundle_index)
+                .map(|domain_bundle_body| {
+                    FraudProofVerificationInfoResponse::DomainBundleBody(domain_bundle_body)
+                }),
         }
     }
+
+    fn derive_bundle_digest(
+        &self,
+        consensus_block_hash: H256,
+        domain_id: DomainId,
+        bundle_body: Vec<OpaqueExtrinsic>,
+    ) -> Option<H256> {
+        let domain_runtime_code = {
```

Comment:
Use the actual function that gets the domain runtime code so there is only one function is used to fetch the runtime code